### PR TITLE
Multi lang support 

### PR DESCRIPTION
**Pull request by BHess2653**
Currently working but is not using the database. This is not ready for release at the moment but is ready for review any feedback to help guide this feature toward the next release will help.

- Migrations & Seeders(commented out in first_run.sh) are made
- process.env.LANG added to gulpfile

### CODE REVIEW

File: bot/client.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ const Localize = require('localize');
...
   help: {
-    en: 'v1.4.0 Discovered Commands:\n\n\t**<> - Required Item\t\t[] - Optional Item**',
-    es: 'v1.4.0 Comandos descubiertos:\n\n\t** <> - Artículo requerido\t\t[] - Artículo opcional**',
-    al: 'v1.4.0 lakwnd awkjjbz:\n\n\t** <> - yfigijgkjg jhckjfiuf\t\t[] - hvkjfnkfcjh yfifkjvk**',
+    EN: 'v1.4.0 Discovered Commands:\n\n\t**<> - Required Item\t\t[] - Optional Item**',
+    ES: 'v1.4.0 Comandos descubiertos:\n\n\t** <> - Artículo requerido\t\t[] - Artículo opcional**',
+    AL: 'v1.4.0 lakwnd awkjjbz:\n\n\t** <> - yfigijgkjg jhckjfiuf\t\t[] - hvkjfnkfcjh yfifkjvk**',
   },
 });
 myLocale.setLocale(process.env.LANG);
```
Comment (mattpezzente) [inline bot/client.js]: Set a fallback for this in case a .env does not have a LANG set by default.

**Review by mattpezzente — CHANGES_REQUESTED** [review]
Overall looks good, I could not think of another way to implement this, as I know doing live translations would be a large undertaking in itself. I know that this is still in progress, but definitely make sure the user is able to set the language if possible.

Lastly, just make sure to add comments for readability.

**Comment by BHess2653** [comment]
This is now ready for release 2 and needs more code reviews.

- messages in en, es, and al are now stored in the bot/locale/messages.json
- added messages.json to controllers
- fixed typo in !addRoles help message from Rolea to Roles

File: readme.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ Create an ```.env``` file with the following sensitive information. Replace the
 ```
 NODE_ENV=development
 DEBUG_MODE=3
+LANG=en
```
Comment (brennondenny) [inline readme.md]: Consider Changing all instances on LANG to LANGUAGE so it will be easily identified in the codebase and the .ENV file

**Review by brennondenny — CHANGES_REQUESTED** [review]
Functionality works with the different commands in message.json, make sure to add LANGUAGE to the README's .ENV setup guide.

File: readme.md (hunk 2 above)
Comment (BHess2653) [reply readme.md]: ok changes have been made.

**Review by brennondenny — APPROVED** [review]

**Comment by BHess2653** [comment]
Updated messages.json with all messages from controllers.
Ready for Release 2

**Review by mattpezzente — APPROVED** [review]

**Comment by reactivepixel** [comment]
There's a lot of redundancy in how you're calling and passing in the environmental variables. It's an environment variable... It should not need passed into anything. 

Also, adding a further level of organization to your .json file that allows the messages be organized according by controller as well will bring that I line with my expectations. 

Those items said... Good implementation!

**Pull request merged**